<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HogGuiClass</class>
 <widget class="QDialog" name="HogGuiClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>956</width>
    <height>514</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>956</width>
    <height>514</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>956</width>
    <height>514</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HogGui</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>3gf1nk5XvJk9QKP.png</normaloff>3gf1nk5XvJk9QKP.png</iconset>
  </property>
  <widget class="QPushButton" name="pic_pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>图片检测</string>
   </property>
  </widget>
  <widget class="QPushButton" name="video_pushButton">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>视频检测</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>120</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>检测到的人数</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>120</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>420</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>图片路径</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>460</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>视频路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="pic_path_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>420</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>D:/C++&amp;&amp;JAVA&amp;&amp;C&amp;&amp;OTHER/opencv/HOG/test.png</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="video_path_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>F:/Program Files/YouKu/Youku Files/download/zyhlc.flv</string>
   </property>
  </widget>
  <widget class="QLabel" name="show_label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>600</width>
     <height>400</height>
    </rect>
   </property>
   <property name="acceptDrops">
    <bool>false</bool>
   </property>
   <property name="whatsThis">
    <string/>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(109, 109, 109);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="load_pushButton">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>121</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>载入图片/视频</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>61</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>分类器路径</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="xml_lineEdit">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>490</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>E:/sample/resultfinnal.xml</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
   <property name="text">
    <string>检测时间</string>
   </property>
  </widget>
  <widget class="QLCDNumber" name="time_lcdNumber">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>170</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 255, 0);</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>170</y>
     <width>20</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 0, 0);</string>
   </property>
   <property name="text">
    <string> ms</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="sample_train">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>200</y>
     <width>321</width>
     <height>301</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="sample">
    <property name="whatsThis">
     <string>因为图片训练的时候需要一种固定的格式，而在程序中已经被设置为固定的内参数，所以需要对图片做统一的裁剪处理，选定图像区域，生成标准的训练图像样本（包括正样本和负样本）。</string>
    </property>
    <attribute name="title">
     <string>样本生成</string>
    </attribute>
    <widget class="QLineEdit" name="postiveinfo_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/INRIAPerson/70X134H96/Test/pos/pos.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="negetiveinfo_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>50</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/INRIAPerson/Train/neg/neg.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="postive_pic_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/INRIAPerson/70X134H96/Test/pos/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="negetive_pic_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/INRIAPerson/Train/neg/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="target_postive_pic_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>140</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/pos1/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="target_negetive_pic_path_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>221</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/neg1/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="total_lineEdit">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>200</y>
       <width>113</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>10000</string>
     </property>
    </widget>
    <widget class="QPushButton" name="create_pushButton">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>230</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>生成</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>pos.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>neg.txt</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>pos.path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>112</y>
       <width>54</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>neg.path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>target.pos.path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>170</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>target.neg.path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>54</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>total</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="train">
    <property name="whatsThis">
     <string>生成好图片样本之后，开始对图片进行训练，将会生成一个训练良好的分类器，而训练器的参数已经设置好</string>
    </property>
    <attribute name="title">
     <string>训练</string>
    </attribute>
    <widget class="QLineEdit" name="postive_infolineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>10</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/pos1/pos.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="negetive_infolineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>40</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/neg1/neg.txt</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="postive_pathlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>70</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/pos1/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="nengtive_pathlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>100</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/neg1/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="save_pathlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>130</y>
       <width>211</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>E:/sample/</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="postive_numlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>1126</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="negetive_numlineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>190</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>2436</string>
     </property>
    </widget>
    <widget class="QPushButton" name="trainpushButton">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>220</y>
       <width>161</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string>开始训练</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="pic_tatallineEdit">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>41</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>3562</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>postive_info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>negetive_info</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>postive_path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>nengtive_path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>save_path</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>151</y>
       <width>91</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>pic_tatal</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>171</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>postive_num</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>192</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>negetive_num</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>统计结果</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="hoggui.qrc"/>
 </resources>
 <connections/>
</ui>
